canonical-websites/www.ubuntu.com
# Added the Partner portal privacy notice - /legal/data-privacy/partner-portal

## Done

- Added the Partner portal privacy notice
- Added the page to the navigation
- Added the page to the Added the /legal/data-privacy page and updated the [copy doc](https://docs.google.com/document/d/1AlOE2_-BqZNP4hiiuf8kZney8g1zvqAlDYl2u3X6m6I/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/data-privacy/partner-portal](http://0.0.0.0:8001/legal/data-privacy/partner-portal)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1yuh7dL3jjkj21sKJCUhm_z00PKzH4vssY339wbp4U-s/edit)
- See that it is in the navigation and on the section overview page [/legal/data-privacy/](http://0.0.0.0:8001/legal/data-privacy/)

## Issue / Card

Fixes #4842
